css-three-column-layout/styles.css

```diff
@@ -0,0 +1,50 @@
+.container {
+  background-color: #1996da;
+  color: white;
+  font-family: Lato, sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.row {
+  display: flex;
+  max-width: 85%;
```

Comment:
Rather than setting a max width on the row, you should set a max width on the container. That way, you aren't stuck with rows that are always smaller than the container